src/extractor/parser/tokenMergers/commentsMerger.ts

```diff
@@ -0,0 +1,45 @@
+import { GeneralTokenType } from '../generalMapper.js';
+import { MachineType } from '../mergerMachine.js';
+
+enum S {
+  Idle,
+  CommentStart,
+  CommentBlock,
+}
```

Comment:
For local, non-exported enums it may be worth using `const enum` so it's collapsed to just integers at build time.

Reply:
That's cool didn't know about that